### PR TITLE
Updated the README and app version

### PR DESCRIPTION
**Pull request by hardikhdholariya**
- Updated the README and bump the app version to 1.0.0

### CODE REVIEW

File: TA-microsoft-graph-security-score/README.md [hunk 1]
```diff
@@ -10,11 +10,11 @@ The Microsoft Graph Security Score Add-on for Splunk allows users to collect the
 
 
 * Author - CrossRealms International Inc.
-* Version - 1.0.1
-* Build - 1
+* Version - 1.1.0
+* Build - 2
```
Comment (mahirchavda) [inline TA-microsoft-graph-security-score/README.md]: Remove Version and Build line from README.

Comment (hardikhdholariya) [reply TA-microsoft-graph-security-score/README.md]: okay, are we incrementing the build number in the app.conf while generating the build?

Comment (mahirchavda) [reply TA-microsoft-graph-security-score/README.md]: No need to change that. it's okay

Comment (hardikhdholariya) [reply TA-microsoft-graph-security-score/README.md]: Done

**Review by mahirchavda — APPROVED** [review]

File: TA-microsoft-graph-security-score/README.md [hunk 2]
```diff
@@ -115,4 +116,4 @@ SUPPORT
 * Contact - CrossRealms International Inc.
   * US: +1-312-2784445
 * License Agreement - https://d38o4gzaohghws.cloudfront.net/static/misc/eula.html
-* Copyright - Copyright CrossRealms Internationals, 2021
```
Comment (VatsalJagani) [inline TA-microsoft-graph-security-score/README.md:118]: Add your name in the Contributors section.

**Review by VatsalJagani — CHANGES_REQUESTED** [review]

**Review by VatsalJagani — APPROVED** [review]

**Pull request merged**